docs/tfengine/schemas/resources.md

```diff
@@ -233,3 +238,10 @@
 | storage_buckets.retention_policy.retention_period | The period of time, in seconds, that objects in the bucket must be retained and cannot be deleted, overwritten, or archived. The value must be less than 2,147,483,647 seconds. | number | false | - | - |
 | storage_buckets.storage_location | Location to create the storage bucket. Can be defined in global data block. | string | false | - | - |
 | terraform_addons | Additional Terraform configuration for the project deployment. Can be used to support arbitrary resources not supported in the following list. For schema see ./deployment.hcl. | object | false | - | - |
+| workload_identity_configurations | [Module](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/modules/workload-identity) | array() | false | - | - |
+| workload_identity_configurations.cluster_name | Cluster name where the workload is deployed. | string | true | - | - |
+| workload_identity_configurations.google_service_account_id | ID of the google service account the deployment should use to authenticate with other resources. | string | true | - | - |
```

Comment:
Consider giving an example in here, otherwise "ID" is kind of ambiguous.

Reply:
Providing an example here is hard since account ID is any string matching with the regular expression [a-z]([-a-z0-9]*[a-z0-9]) to comply with RFC1035. I also couldn't find examples under the `service_accounts` resource for this.
Instead I linked it with the right document.